src/js/pages/system/UnitsHealthDetail.js

```diff
@@ -47,6 +55,24 @@ class UnitsHealthDetail extends mixin(StoreMixin) {
     UnitHealthStore.fetchUnitNodes(this.props.params.unitID);
   }
 
+  onUnitHealthStoreUnitSuccess() {
+    let isLoading = this.state.isLoading & ~FLAGS.UNIT_LOADED;
```

Comment:
Reasoning about these flags is really hard. Also, it's a pattern I haven't seen anywhere else in the codebase. Additionally I am not even sure I understand the need for these in the first place, being both flags constant values.

I also find `isLoading` to not be of type `bool` very strange, but that could be fixed with a `!!`. Still, I'd like to suggest reconsidering this approach to something less contrived, or to at least have comments to explain what's going on.